h2o-r/h2o-package/R/parse.R

```diff
@@ -127,7 +133,8 @@ h2o.parseRaw <- function(data, pattern="", destination_frame = "", header=NA, se
 #' @seealso \link{h2o.parseRaw}
 #' @export
 h2o.parseSetup <- function(data, pattern="", destination_frame = "", header = NA, sep = "", col.names = NULL, col.types = NULL,
-                           na.strings = NULL, parse_type = NULL, chunk_size = NULL, decrypt_tool = NULL, skipped_columns=NULL) {
+                           na.strings = NULL, parse_type = NULL, chunk_size = NULL, decrypt_tool = NULL, skipped_columns=NULL,
+                           custom_non_data_line_markers=NULL) {
```

Comment:
This function has spaces around `=` operator. Some functions have it, some don't. The style is not really unified. To be consistent with function's header, let's change it.